projects/aca-shared/src/lib/shared.module.ts

```diff
@@ -32,10 +32,11 @@ import { MatButtonModule } from '@angular/material/button';
 import { MatIconModule } from '@angular/material/icon';
 import { MatDialogModule } from '@angular/material/dialog';
 import { TranslateModule } from '@ngx-translate/core';
+import { BrowserModule } from '@angular/platform-browser';
```

Comment:
why this is needed as Shared? can you move it to the places/modules where it is needed

Reply:
moved to OpenInAppModule